### ISSUE TITLE
Cannot display log messages from server action

### ISSUE THREAD

**Issue by guidev224** [odoo/odoo#22395]
Impacted versions: 8.0

Steps to reproduce: Use the log object within a server action like log("My message")

Current behavior: name log is not defined

Expected behavior: Display log message from server action

Video/Screenshot link (optional):

@Yenthe666 any help?

**Comment by Yenthe666** [comment]
Hi @guidev224,

Thanks a lot for your bug report but Odoo 8 is no longer supported at this time.
Anyways, you're not using the logging correct. You have to import the logger and then use it:

```
#Import logger
import logging
#Get the logger
_logger = logging.getLogger(__name__)

	_logger.debug("Use _logger.debug for debugging purposes, nothing else.")
	_logger.info("Use _logger.info for information messages. This is used for notifying about something important.")
	_logger.warning("Use _logger.warning for minor issues, which will not crash your module.")
	_logger.error("Use _logger.error to report a failed operation.")
	_logger.critical("Use _logger.critical for critical message: when this goes off the module will not work anymore.")
```

Regards,
Yenthe

**Comment by guidev224** [comment]
Hi @Yenthe666 ,
Thanks for your quick response. My aim is too display a log message from a server action, as you may know Odoo (at least the v8) does not allow import within a server action. That why I think they have made some libraries directly available for the user without having to import them on server actions. 
Here is seemingly the available libraries for the v8 according to the content of the Python Code tab in the server action form.
**Available locals:**
<ul>
<li>- time, datetime, dateutil: Python libraries</li>
<li>- env: Odoo Environement</li>
<li>- model: Model of the record on which the action is triggered</li>
<li>- object: Record on which the action is triggered if there is one, otherwise None</li>
<li>- workflow: Workflow engine</li>
<li>- Warning: Warning Exception to use with raise</li>
<li>To return an action, assign: action = {...}</li>
</ul>
PS: I was reading a book where the author said that there is log object available for displaying log message from server actions.

**Comment by Yenthe666** [comment]
Hmm I'll let @mart-e double check this, perhaps he can provide more feedback. As it is for V8, which is no longer supported, there is nothing we can/will change if it doesn't work though.

**Comment by guidev224** [comment]
Okay, thanks.

**Comment by mart-e** [comment]
Hello,

Indeed this feature was added in 9.0 at 7d0eb114e3b2cf42af0fe11a5c12839c5e3a6a57 but is not available in 8.0

**Comment by guidev224** [comment]
Thanks mart-e
